test/e2e/workload_clusters_test.go

```diff
@@ -140,3 +141,32 @@ func TestDockerUpgradeWorkloadClusterWithFlux(t *testing.T) {
 		provider.WithProviderUpgradeGit(),
 	)
 }
+
+func TestCloudStackKubernetes121WorkloadClusterDemo(t *testing.T) {
+	t.Skip("Skipping CloudStack in CI/CD")
```

Comment:
Any reason we are skipping this specifically and not the others?

Reply:
This would require kube-vip or some load balancer to be natively supported but that's not currently an option with CAPC v0.4.1. It will be unblocked with CAPC v0.4.2

Reply:
Can we add a comment mentioning this, or not have this test until we check that code in?